sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -1252,7 +1252,7 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
           <zeebe:output source="=if studentDataGovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataGovernmentFundingCosts" target="studentDataGovernmentFundingCosts" />
           <zeebe:output source="=if studentDataNongovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataNongovernmentFundingCosts" target="studentDataNongovernmentFundingCosts" />
           <zeebe:output source="=if studentDataParentVoluntaryContributionsCosts = null&#10;then 0&#10;else&#10;  studentDataParentVoluntaryContributionsCosts" target="studentDataParentVoluntaryContributionsCosts" />
-          <zeebe:output source="=if studentPDStatus = null&#10;then false&#10;else&#10;  studentPDStatus" target="studentPDStatus" />
+          <zeebe:output source="=if studentDataApplicationPDPPDStatus = &#34;yes&#34;&#10;then true&#10;else&#10;  false" target="calculatedDataApplicationPDPPDStatus" />
```

Comment:
`calculatedDataApplicationPDPPDStatus` declaration is happening alongside the Camunda task "Set values for null attributes", which does not seem right IMO.
I would prefer to have a specific Camunda task to have this calculation done, similar to the "Define Total Income".

![image](https://github.com/bcgov/SIMS/assets/61259237/cd647d93-7425-49e3-8480-be0fa2de8d26)